webapp/package.json

```diff
@@ -38,7 +38,7 @@
   "scripts": {
     "start": "DISABLE_ESLINT_PLUGIN=true craco start",
     "startConcurrently": "concurrently \"craco start\"",
-    "build": "craco build",
+    "build": "DISABLE_ESLINT_PLUGIN=true craco build",
```

Comment:
Is it possible to do so in .env? This wouldn't work on windows. Maybe we should add Husky to run eslint as precommit hook.

Reply:
Ok, I've put it to .env.production